datalad/interface/clean.py

```diff
@@ -56,32 +57,44 @@ class Clean(Interface):
       $ datalad clean
     """
 
+    result_renderer = 'tailored'
+
     _params_ = dict(
         dataset=Parameter(
             args=("-d", "--dataset"),
             doc="""specify the dataset to perform the clean operation on.  If
                 no dataset is given, an attempt is made to identify the dataset
                 in current working directory""",
             constraints=EnsureDataset() | EnsureNone()),
-        # TODO:  --info  -- which performs dry run just summarizing what is to be cleaned up
+        info=Parameter(
+            args=("-i", "--info"),
+            doc="""Report on cleanable locations - not actually cleaning up
+            anything.""",
+            action='store_true'),
```

Comment:
Thinking about this more, I feel that adopting an already established pattern could have benefits here. `run` uses `run [dry-run]` as action specification in result records. It also uses `--dry-run <mode>` as an argument, not `--info`. I cannot immediately come up with a scenario for more than a boolean flag, but it would still make sense to adopt the naming from my POV. Ultimately it is pretty much a dry-run.

Reply:
No strong opinion on my end. I'm fine with `--dry-run`.

Reply:
Done